fhir-model/src/main/java/com/ibm/fhir/model/util/XMLSupport.java

```diff
@@ -226,15 +226,8 @@ private static XMLOutputFactory createXMLOutputFactory() {
 
     private static TransformerFactory createTransformerFactory() {
         try {
-            boolean isSet = System.getProperty(PROP_TRANSFORMER_FACTORY) != null;
-            if (!isSet) {
-                System.setProperty(PROP_TRANSFORMER_FACTORY, TRANSFORMER_FACTORY_IMPL);
```

Comment:
we can probably remove the PROP_TRANSFORMER_FACTORY now that its not used here

Reply:
Ah yes...copied it back when I reverted the isSet stuff. I'll take it out

Reply:
Wait...no...it's not there - already gone